backend/tests/unit/core/constraint_validators/test_determiner.py

```diff
@@ -59,6 +59,24 @@ def person_name_node_diff(
                 property_name="unique",
             ),
         ),
+        SchemaUpdateConstraintInfo(
```

Comment:
@ajtmccarty I added this otherwise the following tests started failing
```
backend/tests/unit/core/constraint_validators/test_determiner.py::TestConstraintDeterminer::test_one_attribute_update_node_diff FAILED
backend/tests/unit/core/constraint_validators/test_determiner.py::TestConstraintDeterminer::test_node_property_constraints_included FAILED
```
Please can you double check that this is correct